.github/workflows/monitor-studios.yml

```diff
@@ -0,0 +1,28 @@
+name: Test Public Studios
+on:
+  schedule:
+    cron: '0 0 12 * * ?' # Fire at 12:00 p.m. (noon) every day
+
+jobs:
+  test-and-build:
+    runs-on: it
```

Comment:
you should remove the it to run it in the free runner

Reply:
you can use `runs-on: ubuntu-latest`

Reply:
Thanks! Going to try that.